tests/http/testenv/env.py

```diff
@@ -618,3 +618,25 @@ def make_data_file(self, indir: str, fname: str, fsize: int,
                 i = int(fsize / line_length) + 1
                 fd.write(f"{i:09d}-{s}"[0:remain-1] + "\n")
         return fpath
+
+    def make_data_gzipbomb(self, indir: str, fname: str, fsize: int) -> str:
+        fpath = os.path.join(indir, fname)
+        gzpath = f'{fpath}.gz'
+        varpath = f'{fpath}.var'
+        with open(fpath, 'w') as fd:
+            fd.write('not what we are looking for!\n')
+        count = int(fsize / 1024)
+        args = [
+            f'dd if=/dev/zero bs=1024 count={count} | gzip - >{gzpath}'
```

Comment:
you can use the built in Python gzip module for this instead, would improve Windows compat.